File: test/ayush.dev/scala/pipelines/ScalaInsideProjectDatasetsOnly/code/src/main/scala/io/prophecy/pipelines/scalainsideprojectdatasetsonly/graph/Script_1.scala
package io.prophecy.pipelines.scalainsideprojectdatasetsonly.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scalainsideprojectdatasetsonly.config.Context
import io.prophecy.pipelines.scalainsideprojectdatasetsonly.udfs.UDFs._
import io.prophecy.pipelines.scalainsideprojectdatasetsonly.udfs._
import io.prophecy.pipelines.scalainsideprojectdatasetsonly.udfs.PipelineInitCode._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Script_1 {
  def apply(context: Context, in0: DataFrame): DataFrame = {
    val spark = context.spark
    val Config = context.config
    var out0=in0
    out0
  }

}

File: code/e2e/abhishek/depmanagementpy/pipelines/python_externalgit_pipeline/code/job/graph/Script_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from job.config.ConfigStore import *
from job.udfs.UDFs import *

def Script_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    import pandas as pd
    data = {
"calories" : [420, 380, 390], "duration" : [50, 40, 45]}
    df = pd.DataFrame(data)
    print(df)
    import numpy as np
    arr = np.array([1, 2, 3, 4, 5])
    print(arr)
    out0 = in0

    return out0
